Add docsv2 project to master branch (#10000)
netdata/netdata: docs/configure/invite-collaborate.md

heartbeat. When something goes wrong, you'll be ready to collaboratively troubleshoot complex performance problems from
a single pane of glass.

## Invite new members

Invite new users by clicking on your Space's name in the top navigation, and then **Invite more users**, to open the
invitation pane. Admins manage user permissions and have control over who can access specific Spaces and War Rooms.

![Opening and navigating the invitation
panel](https://user-images.githubusercontent.com/1153921/92025596-a618e680-ed14-11ea-9c1f-a61fdcb8aa4e.png)

Enter their email address and name. They can change this name once they accept your invitation.

Choose which War Rooms you want to add this user to, then click the plus **+** button to add the invitation to the
**New invitations to be sent** queue. Repeat the process with everyone you want to invite to your Space.

When you're ready to send the new invitations you created, hit the **Send** button. Netdata Cloud sends these
invitations and moves them to the **Invitations awaiting response** category.

Your team will receive their email invitations momentarily with a prompt to sign in to join your Space.

## Collaboration with Netdata Cloud

> Netdata Cloud gives teams a single interface to view real-time metrics from across the entire infrastructure. Having all

Reviewer: ```suggestion
Netdata Cloud gives teams a single interface to view real-time metrics across their entire infrastructure. Having all
```
